packages/icons/scripts/build.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const glob = require('glob');
+const SVGO = require('svgo');
+const path = require('path');
+const prettier = require('prettier');
+const camelCase = require('lodash.camelcase');
+
+const svgo = new SVGO({});
+
+const svgPaths = glob.sync('./svg/*');
+const outputDir = './react/';
```

Comment:
Maybe call it `__generated__`  or something, so it's more clear these files are generated? 🤔